swabbie-aws/src/main/kotlin/com/netflix/spinnaker/swabbie/aws/images/AmazonImage.kt

```diff
@@ -27,9 +28,24 @@ data class AmazonImage(
   val ownerId: String?,
   val description: String?,
   val state: String,
+  val blockDeviceMappings: List<AmazonBlockDevice>?,
   override val resourceId: String = imageId,
   override val resourceType: String = IMAGE,
   override val cloudProvider: String = AWS,
   override val name: String?,
   private val creationDate: String?
 ) : AmazonResource(creationDate)
+
+@JsonTypeName("blockDevice")
```

Comment:
This annotation is not necessary here unless we are intending to operate on objects of this kind in the future.

Reply:
Are we seeing failures to marshal an image when `JsonTypeName` is omitted?

Reply:
Good point - I was seeing some failures and was trying to figure out why. I think the root of my problem was that I was using an object mapper in the tests which didn't have the json ignore unknown properties set. I will test out removing these annotations.